src/lib/constants.ts

```diff
@@ -50,6 +50,10 @@ const constants = {
 	bootBlockDevice: '/dev/mmcblk0p1',
 	hostConfigVarPrefix: 'HOST_',
 	migrationBackupFile: 'backup.tgz',
+	// The amount of time to increase the wait by on each failure
```

Comment:
Strictly speaking the backoff is doubled each time starting at this value, ie it'll go 500 -> 1000 -> 2000 -> 4000, rather than 500 -> 1000 -> 1500 -> 2000 which is what this comment implies

Reply:
Updated